### PR TITLE
Toggling Issue in Dark/Light themes 

### LINKED ISSUE #85: Issue when toggling Dark/Light themes
## Expected Behavior
      The font in the textbox, when toggling the website from Black Theme to white theme, should be visible

## Current Behavior and Triggers
 It shows white font, thus on toggling , the font is not visible, unless we edit the data.

## Possible Solution

if we could adjust the font-color on toggling the themes.

## Your Environment

* Version used:
* Environment name and version (e.g. Chrome 39, node.js 5.4):Chrome 74.0.3729.169 (Official Build) (64-bit
* Operating System and version (desktop or mobile): Windows 10 x64, Desktop
* Link to your project:

## Would you like to work on this issue?

No

Images, showing this issue, are attached
image
image

### PR DESCRIPTION
**Pull request by prathmeshkhandelwal1**
Fixes #85 

### Description

The font in the textbox, when toggling the website from Black Theme to the white theme, should be visible

###  Screenshot Before Change:

Screenshot (95)

### Screenshot After Change:
Screenshot (96)

### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Non Functional Requirement
- [x] Follows the code style of this project.
- [x] Tests Cover Changes
- [x] I have performed a self-review of my own code
- [ ] All new and existing tests passed.
- [ ] Documentation

### CODE REVIEW

**Comment by aashutoshrathi** [comment]
@AmanRaj1608 I think requesting review from `iiitv/project-maintainers` is enough.

**Comment by aashutoshrathi** [comment]
@prathmeshkhandelwal1 Can you give us a test deployment link?

**Comment by prathmeshkhandelwal1** [comment]
https://prathmeshkhandelwal1-getmein-web.glitch.me/ @aashutoshrathi

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
Have some issues if I start with light theme

captured

**Comment by prathmeshkhandelwal1** [comment]
Changes done @aashutoshrathi  deployment -link: https://prathmeshkhandelwal1-getmein-web.glitch.me

**Comment by prathmeshkhandelwal1** [comment]
@iiitv/project-maintainers can you plz review it?

**Review by aashutoshrathi — APPROVED** [review]

**Comment by aashutoshrathi** [comment]
Delete commented code and make sure newline at the end of file

**Comment by prathmeshkhandelwal1** [comment]
okk Sir

**Comment by prathmeshkhandelwal1** [comment]
Sir what do you mean by new line at the end of file.. i dont get it

File: public/darkmode.js [hunk 1]
```diff
@@ -7,3 +7,14 @@ $('#toggleTheme').on('change', () => {
...
+// const toggleTheme = document.getElementById("toggleTheme")
+// const input = document.getElementById('email')
+
+// toggleTheme.addEventListener('click',()=>{
+//   if(toggleTheme.checked){
+//     input.style.backgroundColor = "white"
+//   }else{
+//     input.style.background = "black"
+//   }
+// })
```
Comment (thecodepapaya) [inline public/darkmode.js]: @prathmeshkhandelwal1 add a new line character at the end of the file here

**Comment by prathmeshkhandelwal1** [comment]
Done Sir @aashutoshrathi

**Review by AmanRaj1608 — APPROVED** [review]
LGTM ✨

File: public/dist/darkmode.dev.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  }
+}); // const toggleTheme = document.getElementById("toggleTheme")
+// const input = document.getElementById('email')
+// toggleTheme.addEventListener('click',()=>{
+//   if(toggleTheme.checked){
+//     input.style.backgroundColor = "white"
+//   }else{
+//     input.style.background = "black"
+//   }
+// })
```
Comment (thecodepapaya) [inline public/dist/darkmode.dev.js:19]: @prathmeshkhandelwal1 add a new line character at the end of the file here as well

File: public/darkmode.js (hunk 1 above)
Comment (thecodepapaya) [inline public/darkmode.js]: Remove code that has been commented out.

**Review by thecodepapaya — APPROVED** [review]

**Pull request merged**